src/main/java/com/gamegoo/apiPayload/code/status/ErrorStatus.java

```diff
@@ -137,7 +138,9 @@ public enum ErrorStatus implements BaseErrorCode {
     NOTIFICATION_TYPE_NOT_FOUND(HttpStatus.NOT_FOUND, "NOTI401", "잘못된 알림 타입입니다."),
     NOTIFICATION_METHOD_BAD_REQUEST(HttpStatus.BAD_REQUEST, "NOTI402", "알림 생성 메소드 호출이 잘못되었습니다."),
 
-    ;
+    // SOCKET 서버 API 호출 에러
+    SOCKET_API_RESPONSE_ERR(HttpStatus.INTERNAL_SERVER_ERROR, "SOCKET501",
```

Comment:
이 부분 SOCKET_API_RESPONSE_ERR -> SOCKET_API_RESPONSE_ERROR 로 변경하셔도 좋을 것 같습니다!
```suggestion
    SOCKET_API_RESPONSE_ERROR(HttpStatus.INTERNAL_SERVER_ERROR, "SOCKET501",
```